src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurStandardAccountingExtractFileFixture.java

```diff
@@ -37,6 +37,7 @@ private static ConcurStandardAccountingExtractDetailLine buildConcurStandardAcco
         ConcurStandardAccountingExtractDetailLine line = new ConcurStandardAccountingExtractDetailLine();
         line.setJounalDebitCredit(debitCredit);
         line.setJournalAmount(amount);
+        line.setEmployeeGroupId("CORNELL");
```

Comment:
This is a unit test, so it does not necessarily need to use the parameter service to derive the value. However, it would still be a good idea to at least move this to its own constant.